woocommerce/woocommerce
# [Experimental] Product Filters Redesign: Add the Product Filters block

### Changes proposed in this Pull Request:

This PR adds the Product Filters block, which is a wrapper for many other filters such as the: Active Filters, Stock Status Filter, Price Filter, etc.





Closes #46984.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Navigate to Single Product template (Appearance > Editor > Templates > Product Catalog template - or any archive template).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. Search for "Product Filters (Beta)" in the block inserter or browse the available blocks until you find them. Add it to your content.
5. Make sure you can see a heading with the text "Filters" on the top of the block.
6. Make sure you can see all of these inner blocks: Active Filters, Price Filter, Stock Status Filter, Attribute Filter, Rating Filter.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 
Add the Product Filters block

</details>
